src/FSAL/FSAL_CORTXFS/handle.c

```diff
@@ -587,6 +665,8 @@ static fsal_status_t kvsfs_mkdir(struct fsal_obj_handle *dir_hdl,
 		 *  access checks using ACLs are implemented.
 		 */
 
+		perfc_trace_attr(PEA_OTHER_FUNC_CALL, 1);
```

Comment:
Is this attribute being used to track time taken by the interim code?

Reply:
thats correct